README.md

```diff
@@ -1,30 +1,58 @@
-<!--
-  -- Copyright 2020 Contributors to the Parsec project. 
-  -- SPDX-License-Identifier: Apache-2.0
---->
 # Parsec Tool
 
 This repository contains a tool to communicate with the [Parsec
 service](https://github.com/parallaxsecond/parsec) on the command-line.
 
-For demos and to test the Parsec service, you might want to change the Parsec endpoint location.
-For that, set the `PARSEC_SERVICE_ENDPOINT` environment variable to correction endpoint.
+## Getting started
+
+To compile and list the available commands:
+
+```
+$ cargo build
+$ ./target/debug/parsec-tool --help
+```
+
+## Modifying Parsec service endpoint
+
+For demos and to test the Parsec service, you might want to change the Parsec endpoint location. For
+that, set the `PARSEC_SERVICE_ENDPOINT` environment variable to correction endpoint.
 
 To set a Unix Domain Socket Listener endpoint at `/tmp/parsec.sock`:
 
 ```
 $ export PARSEC_SERVICE_ENDPOINT=unix:/tmp/parsec.sock
 ```
 
+## Modifying logging output
+
+You can set the `RUST_LOG` environment variable to modify the logging outpout. See [the
+documentation](https://docs.rs/env_logger/0.8.3/env_logger/index.html) for more information.
+
+## Data format
+
+Unless specified otherwise below, the data format expected by the commands is the same as describe
+in the [Parsec
+Book](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/index.html). The
+`--help` option of commands might give more information about the expected format.
+
+The ECC public keys are formatted using the ASN.1 representation `Ecdsa-Sig-Value` described in [RFC
```

Comment:
```suggestion
ECDSA signatures are formatted using the ASN.1 representation `Ecdsa-Sig-Value` described in [RFC
```

Also, it's worth saying that `export_public_key` formats them in PEM

Reply:
oops